infrastructure/k8s/services/secrets.yaml

```diff
@@ -0,0 +1,29 @@
+project_name: "taskcluster-secrets"
+secrets:
+  azure_account: ".Values.tfAzureAccount"
+  azure_crypto_key: ".Values.azureCryptoKey"
+  azure_signing_key: ".Values.azureSigningKey"
+  azure_table_name: "Secrets"
+  force_ssl: "false"
+  monitoring_enable: "true"
+  node_env: "production"
+  publish_metadata: "false"
+  pulse_password: ".Values.pulsePassword"
+  pulse_username: ".Values.pulseUsername"
```

Comment:
Can you rename these to  notifyPulsePassword and notifyPulseUsername?  I think each service has their own.

Also, can you add generating the taskcluster-notify user to to create-rabbitmq-users.sh?

Tangentially, it's a little odd that their terraform has the secrets service using the rabbitmq credentials created for the notification service. You could check if this was an accident and we should be creating a new user for this service instead.

(Note: Looking at the

Reply:
OK, I'll make those modifications. We probably do want individual sets of credentials. 

Also, "(Note: Looking at the" ?

Reply:
Heh, I forget what I was going to write.